app/controllers/plants_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PlantsController < ApplicationController
+
+  def index
+    @plant = current_user.plants.all
+  end
+    
+  def new
+    @plant = Plant.new
+  end
+    
+  def create
+    @plant = current_user.plants.create(plant_params)
```

Comment:
do you need to use an instance variable here?

Reply:
Yes if only because it shortens the syntax on the following if statement in the same file:

if @plant.persisted?
      flash[:notice] = 'Your plant is now for sale!'
      redirect_to user_path(current_user)
    else
      flash[:alert] = 'Invalid entry.'
      redirect_to new_plant_path
    end

Do we need to remove it?

Reply:
Removing the @ sign